apps/camera/index.html

```diff
@@ -10,7 +10,10 @@
     <link rel="localization" href="locales/camera.{locale}.properties">
     <link rel="localization" href="/shared/locales/date/date.{locale}.properties">
     <link rel="localization" href="/shared/locales/media/media.{locale}.properties">
+    <link rel="localization" href="/shared/elements/gaia_menu/locales/gaia_menu.{locale}.properties">
     <script defer src="/shared/js/l10n.js"></script>
+    <script defer src="shared/js/component_utils.js"></script>
+    <script defer src="shared/elements/gaia_menu/script.js"></script>
```

Comment:
The camera app uses Bower to manage its dependencies on web components, so you'll need to edit bower.json to import the upstream gaia_menu component rather than using the one in shared. I'd be somewhat surprised if component_utils is not already included somehow.

Also, we're very sensitive to the startup time of our apps.  Since there is no menu displayed when the app starts up, we'd need this file to be loaded when it is needed rather than loaded at startup.  The camera app uses requirejs to do its loading.